### ISSUE TITLE
[BUG]训练自定义数据集时在数据增强部分挂掉，显示Error: Blocking queue is killed because the data reader raises an exception

### ISSUE THREAD

**Issue by yangnan456** [PaddlePaddle/PaddleDetection#5667]
**PaddleDetection team appreciate any suggestion or problem you delivered~**

## Checklist:

1. 查找历史相关issue寻求解答/I have searched related issues but cannot get the expected help.
2. 翻阅[FAQ](https://paddledetection.readthedocs.io/FAQ.html) /I have read the [FAQ documentation](https://paddledetection.readthedocs.io/FAQ.html) but cannot get the expected help.
3. 确认bug是否在新版本里还未修复/The bug has not been fixed in the latest version.

## 描述问题/Describe the bug
在训练自定义数据集时出现Error: Blocking queue is killed because the data reader raises an exception，此外还有
、
![image](https://user-images.githubusercontent.com/99865141/162910353-6ef3344d-9e6d-459a-bbc1-33eeb9c1321b.png)
![image](https://user-images.githubusercontent.com/99865141/162910412-24366ef9-7d99-418f-8c80-92acf06da324.png)
![image](https://user-images.githubusercontent.com/99865141/162910499-fa3eba84-3a6b-4a34-9173-82942665f84b.png)
![image](https://user-images.githubusercontent.com/99865141/162910574-37c919f7-69e7-4b31-ab88-d0296c7bd040.png)
![image](https://user-images.githubusercontent.com/99865141/162910663-70943cf3-8023-4f37-ac5a-147ed4880ad6.png)
![image](https://user-images.githubusercontent.com/99865141/162910708-9c416c81-b54c-4bb8-99a6-a47d0ec9bb2d.png)

## 复现/Reproduction

1. 您使用的命令是？/What command or script did you run?
 python PaddleDetection-release-0.5/tools/train.py -c PaddleDetection-release-0.5/configs/ssd/ssd_mobilenet_v1_voc.yml 
```none
请填写命令/A placeholder for the command.
```
2. 您是否更改过代码或配置文件？您是否理解您所更改的内容？还请您提供所更改的部分代码。/Did you make any modifications on the code or config? Did you understand what you have modified? Please provide the codes that you modified.
更改的代码有配置文件ssd——mobilnet——voc。yxml
architecture: SSD
pretrain_weights: https://paddlemodels.bj.bcebos.com/object_detection/ssd_mobilenet_v1_coco_pretrained.tar
use_gpu: true
max_iters: 20000
snapshot_iter: 180
log_iter: 30
metric: VOC
map_type: 11point
save_dir: output
weights: output/ssd_mobilenet_v1_voc/model_final
# 20(label_class) + 1(background)
num_classes: 10

SSD:
  backbone: MobileNet
  multi_box_head: MultiBoxHead
  output_decoder:
    background_label: 0
    keep_top_k: 200
    nms_eta: 1.0
    nms_threshold: 0.45
    nms_top_k: 400
    score_threshold: 0.01

MobileNet:
  norm_decay: 0.
  conv_group_scale: 1
  conv_learning_rate: 0.1
  extra_block_filters: [[256, 512], [128, 256], [128, 256], [64, 128]]
  with_extra_blocks: true

MultiBoxHead:
  aspect_ratios: [[2.], [2., 3.], [2., 3.], [2., 3.], [2., 3.], [2., 3.]]
  base_size: 300
  flip: true
  max_ratio: 90
  max_sizes: [[], 150.0, 195.0, 240.0, 285.0, 300.0]
  min_ratio: 20
  min_sizes: [60.0, 105.0, 150.0, 195.0, 240.0, 285.0]
  offset: 0.5

LearningRate:
  schedulers:
  - !PiecewiseDecay
    milestones: [10000, 15000, 20000, 25000]
    values: [0.001, 0.0005, 0.00025, 0.0001, 0.00001]

OptimizerBuilder:
  optimizer:
    momentum: 0.0
    type: RMSPropOptimizer
  regularizer:
    factor: 0.00005
    type: L2

TrainReader:
  inputs_def:
    image_shape: [3, 320, 240]
    fields: ['image', 'gt_bbox', 'gt_class']
  dataset:
    !VOCDataSet
    anno_path: train.txt
    dataset_dir: /home/aistudio/work/src/ImageSet
    use_default_label: false
  sample_transforms:
  - !DecodeImage
    to_rgb: true
  - !RandomDistort
    brightness_lower: 0.875
    brightness_upper: 1.125
    is_order: true
  - !RandomExpand
    fill_value: [127.5, 127.5, 127.5]
  - !RandomCrop
    allow_no_crop: false
  - !NormalizeBox {}
  - !ResizeImage
    interp: 1
    target_size: 300
    use_cv2: false
  - !RandomFlipImage
    is_normalized: true
  - !Permute {}
  - !NormalizeImage
    is_scale: false
    mean: [127.5, 127.5, 127.5]
    std: [127.502231, 127.502231, 127.502231]
  batch_size: 64
  shuffle: true
  drop_last: true
  worker_num: 8
  bufsize: 16
  use_process: true

EvalReader:
  inputs_def:
    image_shape: [3, 320, 240]
    fields: ['image', 'gt_bbox', 'gt_class', 'im_shape', 'im_id', 'is_difficult']
  dataset:
    !VOCDataSet
    anno_path: val.txt
    dataset_dir: /home/aistudio/work/src/ImageSet
    use_default_label: false
  sample_transforms:
  - !DecodeImage
    to_rgb: true
  - !NormalizeBox {}
  - !ResizeImage
    interp: 1
    target_size: 300
    use_cv2: false
  - !Permute {}
  - !NormalizeImage
    is_scale: false
    mean: [127.5, 127.5, 127.5]
    std: [127.502231, 127.502231, 127.502231]
  batch_size: 32
  worker_num: 8
  bufsize: 16
  use_process: false

TestReader:
  inputs_def:
    image_shape: [3,320,240]
    fields: ['image', 'im_id', 'im_shape']
  dataset:
    !ImageFolder
    anno_path: /home/aistudio/work/src/ImageSet/label_list.txt
    use_default_label: false
  sample_transforms:
  - !DecodeImage
    to_rgb: true
  - !ResizeImage
    interp: 1
    max_size: 0
    target_size: 300
    use_cv2: true
  - !Permute {}
  - !NormalizeImage
    is_scale: false
    mean: [127.5, 127.5, 127.5]
    std: [127.502231, 127.502231, 127.502231]
  batch_size: 1

3. 您使用的数据集是？/What dataset did you use?
自定义数据集，voc格式
![image](https://user-images.githubusercontent.com/99865141/162911194-28f30ab0-f400-4e1d-aa8c-0aa8c1e52e45.png)
![image](https://user-images.githubusercontent.com/99865141/162911243-1466e064-9354-4a28-8d16-f418aac58903.png)
![image](https://user-images.githubusercontent.com/99865141/162911294-267a3ffc-e30d-4624-96a4-8453d87b79e6.png)
![image](https://user-images.githubusercontent.com/99865141/162911363-b7d22049-1dbe-4453-91df-5834741165ea.png)

4. 请提供您出现的报错信息及相关log。/Please provide the error messages or relevant log information.

## 环境/Environment
1. 请提供您使用的Paddle和PaddleDetection的版本号/Please provide the version of Paddle and PaddleDetection you use：
Paddle1.8.4
Paddledetection0.5
python3.7
需要部署到edgeboardz3上
2. 如您在使用PaddleDetection的同时还在使用其他产品，如PaddleServing、PaddleInference等，请您提供其版本号/ Please provide the version of any other related tools/products used, such as the version of PaddleServing and etc：

3. 请提供您使用的操作系统信息，如Linux/Windows/MacOS /Please provide the OS information, e.g., Linux：
win10
4. 请问您使用的Python版本是？/ Please provide the version of Python you used.

5. 请问您使用的CUDA/cuDNN的版本号是？/ Please provide the version of CUDA/cuDNN you used.

如果您的issue是关于安装或环境，您可以先查询[安装文档](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/docs/tutorials/INSTALL_cn.md)尝试解决~

If your issue looks like an installation issue / environment issue,
please first try to solve it yourself with the instructions in
https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/docs/tutorials/INSTALL.md

**Comment by nemonameless** [comment]
建议换最新release/2.4分支，以及Paddle2.2.2去运行。如果你仍然要用这个版本，建议先跑下VOC数据集，确保没问题再跑自己做的数据集，大概率是数据集制作的问题建议仔细自查。
